recipes/splink/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "splink" %}
+{% set version = "0.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2d597bee5ff921658867ce87581629c6cc861711cb7ff85ed07778c2e927b2f7
+
+build:
```

This looks like it can be noarch python?
```suggestion
build:
  noarch: python
```